### ISSUE TITLE
From RNN to Transformers

### ISSUE THREAD

**Issue by jinmang2** [jinmang2/bring_it_on#5]
RNN에서 Transformers까지 모든 수학적, 기반 내용 및 관련된 논문 증명 모조리 읽고 내 것으로 만든다.

그러고 나선 Korean Transformers 라이브러리를 꾸며야지

**Comment by jinmang2** [comment]
https://www.mitpressjournals.org/doi/pdf/10.1162/neco.1989.1.2.270

https://m.blog.naver.com/PostView.nhn?blogId=sooftware&logNo=221790750668&categoryNo=0&proxyReferer=https:%2F%2Fwww.google.com%2F

http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.48.9879&rep=rep1&type=pdf

https://arxiv.org/abs/1312.6026

https://arxiv.org/abs/1506.03099

http://egrcc.github.io/docs/Machine-Learning-9-Recurrent-Neural-Network.pdf

https://crl.ucsd.edu/~elman/Papers/fsit.pdf

https://arxiv.org/pdf/1211.5063.pdf

https://mmuratarat.github.io/2019-02-07/bptt-of-rnn

https://aikorea.org/blog/rnn-tutorial-3/

http://www.wildml.com/2015/10/recurrent-neural-networks-tutorial-part-3-backpropagation-through-time-and-vanishing-gradients/

https://arxiv.org/abs/1211.5063

https://arxiv.org/abs/1610.02583

**Comment by jinmang2** [comment]
https://github.com/jadore801120/attention-is-all-you-need-pytorch/issues/8

https://medium.com/platfarm/%EC%96%B4%ED%85%90%EC%85%98-%EB%A9%94%EC%BB%A4%EB%8B%88%EC%A6%98%EA%B3%BC-transfomer-self-attention-842498fd3225

https://medium.com/inside-machine-learning/what-is-a-transformer-d07dd1fbec04

https://papers.nips.cc/paper/2019/file/2f4fe03d77724a7217006e5d16728874-Paper.pdf

https://arxiv.org/pdf/2002.04745.pdf

https://www.aclweb.org/anthology/2020.coling-main.320.pdf

**Comment by jinmang2** [comment]
https://arxiv.org/pdf/1607.03474.pdf

**Comment by jinmang2** [comment]
https://ratsgo.github.io/deep%20learning/2017/10/10/RNNsty/

**Comment by jinmang2** [comment]
https://stackoverflow.com/questions/44238154/what-is-the-difference-between-luong-attention-and-bahdanau-attention

**Comment by jinmang2** [comment]
https://machinelearningmastery.com/teacher-forcing-for-recurrent-neural-networks/

https://github.com/jadore801120/attention-is-all-you-need-pytorch/issues/8

https://arxiv.org/pdf/1906.07651.pdf
